tsup.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'tsup';
+import { allLocales } from './src';
+
+export default defineConfig({
+  entry: [
+    'src/index.ts',
+    ...Object.keys(allLocales).map((locale) => `src/locale/${locale}.ts`),
+  ],
+  outDir: 'dist',
+  clean: true,
+  format: ['esm', 'cjs'],
+  target: ['es2019', 'node14.17'],
```

Comment:
when we want to merge this in v9 instead of v8.x, then this needs to be
```suggestion
  target: ['es2022', 'node18'],
```